nifi-extension-bundles/nifi-poi-bundle/nifi-poi-services/src/test/java/org/apache/nifi/excel/TestExcelRecordReader.java

```diff
@@ -141,15 +140,15 @@ public void testMultipleRecordsSingleSheet() throws MalformedRecordException {
                 .build();
 
         ExcelRecordReader recordReader = new ExcelRecordReader(configuration, getInputStream(DATA_FORMATTING_FILE), logger);
-        List<Record> records = getRecords(recordReader, false, false);
+        List<Record> records = getRecords(recordReader, false);
 
         assertEquals(9, records.size());
     }
 
     private RecordSchema getDataFormattingSchema() {
         final List<RecordField> fields = Arrays.asList(
                 new RecordField("Numbers", RecordFieldType.DOUBLE.getDataType()),
-                new RecordField("Timestamps", RecordFieldType.DATE.getDataType()),
+                new RecordField("Timestamps", RecordFieldType.STRING.getDataType()),
```

Comment:
Is this type change just to simplify the test, with the understanding that `TestExcelReader` covers the various field types?

Reply:
Yes